extra/utils.py

```diff
@@ -176,10 +176,11 @@ def fake_torch_load(b0):
     key_real[key_lookup.index(k)] = v[0]
 
   # read in the actual data
-  for storage_type, obj_size, tensor, np_shape, np_strides in key_real:
+  for storage_type, obj_size, tensor, np_shape, np_strides, storage_offset in key_real:
     ll = struct.unpack("Q", fb0.read(8))[0]
     assert ll == obj_size, f"size mismatch {ll} != {obj_size}"
-    load_single_weight(tensor, fb0, np_shape, np_strides, storage_type)
+    assert storage_offset == 0, "not implemented"
+    load_single_weight(tensor, fb0, np_shape, np_strides, storage_type, None)
```

Comment:
Why is mmap_allowed=None?

Reply:
it's `storage_offset`. The model from the test actually breaks old torch format loading, maybe someone else fixes

It fails even before the assert I added, here:
https://github.com/geohot/tinygrad/blob/e28bd11ff142b67ad311e5f9e40a7d66da20618b/extra/utils.py#L174-L176